analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/OperationLightupGenerator.cs

```diff
@@ -1047,9 +1149,17 @@ public PropertyData(XElement node)
                     "InterpolatedStringArgumentPlaceholderKind" => true,    // Sonar: Skipped because it's not available
                     "LoopKind" => true,
                     "PlaceholderKind" => true,
```

Comment:
I can see in the StyleCop.xml a few more elements:
```cs
"BinaryOperatorKind" => true,
"CaptureId" => true,
"UnaryOperatorKind" => true,
```
where they removed on purpose?

Reply:
Yes, all of them. If I let them ignored, we will have compilation errors since we use the generated types.